.gitignore

```diff
@@ -7,3 +7,6 @@ cmake-build-debug
 .DS_Store
 .swiftpm
 .vscode
+
+# Ignore XCode project local user data
+Test/**/xcuserdata
```

Comment:
Can we add a proper gitignore generated by some tool like https://www.toptal.com/developers/gitignore/api/xcode